### ISSUE TITLE
Latest commits?

### ISSUE THREAD

**Issue by stoffern** [jeanpan/react-native-camera-roll-picker#11]
@jeanpan 

Batchsize:
Why use `batchSize`, it makes no sense?

Retriving 5 images or 4000 images in a fetch has a difference of 93 ns.
With `SGListView` it will only render items that are in the view and flush them again if they are out of bounds.

Facebook use batchSize because they render all the items in the list at load, and rendering 4000 at once will require batchSize

Example:
Why add Android, and IOS folders to the repo? 
I cleaned them out as a exampe only need to have the components explaning the usage. not full repo of a React Native application.?
Just put it all into the `.gitignore`

**Comment by jeanpan** [comment]
@stoffern 

I put Example back because it's useful for testing, and it's not included in the npm module. I don't think it should be removed. Regarding `batchSize`, I will take a look at `SGListView` and figure out what's the difference between `SGListView` and `ListView`. Thanks for your suggestion. 

Moreover, there is a problem in your #7 PR which is the selectedMarker not shown when loading more images ( I tested it with 25 images ). I think it's caused by the listview didn't rerender. I fixed this problem and I also added loading indicator.
